docs/core/testing/unit-testing-platform-integration-dotnet-test.md

```diff
@@ -56,23 +56,32 @@ By default, VSTest is used to run `Microsoft.Testing.Platform` tests. You can en
 </Project>
 ```
 
-In this mode, you can supply extra parameters that will be utilized to call the testing application as follows:
+In this mode, you can supply extra parameters that are used to call the testing application in one of the following ways:
 
-1. Beginning with `Microsoft.Testing.Platform` version 1.4 (included with MSTest version 3.6), options can be added after the double dash `--`:
+- Beginning with `Microsoft.Testing.Platform` version 1.4 (included with MSTest version 3.6), you can options after the double dash `--` on the command line:
 
     ```dotnetcli
     dotnet test -- --minimum-expected-tests 10
     ```
 
-1. By using the `TestingPlatformCommandLineArguments` MSBuild property:
+- By using the `TestingPlatformCommandLineArguments` MSBuild property on the command line:
 
     ```dotnetcli
     dotnet test -p:TestingPlatformCommandLineArguments=" --minimum-expected-tests 10 "
```

Comment:
Is the value supposed to be surrounded by spaces?

Reply:
According to this, no: https://github.com/nogic1008/WritableOptions.Net/blob/main/.github/workflows/dotnet.yml#L62

Reply:
Indeed, that's not needed.